index.html

```diff
@@ -177,10 +183,21 @@ <h2>Other</h2>
 			<span id="tooltiptext" class="tooltiptext"></span>
 		</div>
 		<div id="copiedTooltip" class="tooltip"><span class="tooltiptext">Copied!</span></div>
-		<div id="export">
+		<div id="export" class="popup">
 			<p>The goals of the current sheet as JSON:</p>
-			<textarea readonly="readonly"></textarea>
-			<a href="javascript:hideGoalExport()">Hide</a>
+			<textarea readonly="readonly" id="goals_in_json"></textarea>
+			<div class="popup-button-group">
+				<button class="button" title="Copy JSON to Clipboard" onclick='copyToClipboard("goals_in_json", event, true);'>Copy JSON</button>
+				<button class="button" onclick="closePopupByButton(this);">Close</button>
+			</div>
+		</div>
+		<div id="urlWithProgress" class="popup" data-keep-options-open="true">
```

Comment:
Element `#urlWithProgress` doesn't contain any URLs itself. It is a popup explaining how `&progress` works. I suggest renaming it to `progress_notice` by analogy with `version_notice` and `version_notice_unstable`.